pypulseq/Sequence/calc_grad_spectrum.py

```diff
@@ -0,0 +1,180 @@
+from typing import Tuple, List
+
+import numpy as np
+from scipy.signal import spectrogram
+from matplotlib import pyplot as plt
+
+
+def calculate_gradient_spectrum(
+        obj,
+        max_frequency: float = 2000,
+        window_width: float = 0.05,
+        frequency_oversampling: float = 3,
+        time_range: List[float] = None,
```

Comment:
should be `List[float] | None` or `Union[List[float], None]` if it defauls to None

Reply:
Done